ui/apps/platform/src/services/VulnerabilitiesService.js

```diff
@@ -93,8 +93,20 @@ export function exportCvesAsCsv(fileName, workflowState, cveType) {
         return { ...{ [key]: fullEntityContext[key] } };
     }, {});
 
+    const currentSearchState = workflowState.getCurrentSearchState();
+
+    // TODO: remove after Postgres is required for all installations
```

Comment:
I have one concern about our TODOs. We add them but I know sometimes we can forget about them later on. My suggestion is that we should couple our TODOs with an actual Jira ticket, under whatever Epic is appropriate, and track clean-up work as well. That way we don't accumulate tech debt

That also makes me think, maybe we should go through our TODOs sometime and try to track some clean up or updates that we might have forgotten about

cc: @vjwilson @alwayshooin @pedrottimark @dvail @bradr5

Reply:
Good ideas.

I this case, I intentionally fashioned the conditional so that it won't run after the transition to Postgres.